src/components/SEP/Proposals/SEPProposalsAndAssignmentsTable.tsx

```diff
@@ -44,6 +44,64 @@ type SEPProposalsAndAssignmentsTableProps = {
   selectedCallId: number;
 };
 
+const getGradesFromAssignments = (assignments: SEPProposalAssignmentType[]) =>
+  assignments?.map((assignment) => assignment.review?.grade) ?? [];
+
+const SEPProposalColumns = [
+  { title: 'ID', field: 'proposal.proposalId' },
+  {
+    title: 'Title',
+    field: 'proposal.title',
+  },
+  {
+    title: 'Status',
+    field: 'proposal.status.name',
+  },
+  {
+    title: 'Date assigned',
+    field: 'dateAssigned',
+    render: (rowData: SEPProposalType): string =>
+      dateformat(new Date(rowData.dateAssigned), 'dd-mmm-yyyy HH:MM:ss'),
+  },
+  {
+    title: 'Reviewers',
+    field: 'assignments.length',
+    emptyValue: '-',
+  },
+  {
+    title: 'Average grade',
+    render: (rowData: SEPProposalType): string => {
+      const avgGrade = average(
+        getGradesFromAssignments(rowData.assignments ?? []) as number[]
```

Comment:
Instead of casting as `number[]` everywhere you could add a return type to `getGradesFromAssignment` to `number[]`